src/app/editor/contextView.js

```diff
@@ -159,7 +168,14 @@ class ContextView {
       <span class=${css.referencesnb}>${references}</span>
       <i data-action='previous' class="fa fa-chevron-up ${css.jump}" aria-hidden="true" onclick=${jump}></i>
       <i data-action='next' class="fa fa-chevron-down ${css.jump}" aria-hidden="true" onclick=${jump}></i>
+        ${showGasEstimation()}
     </div>`
+
+    function showGasEstimation () {
+      var estimatedGas = self._api.contextualListener.gasEstimation(node) + ' gas'
+      if (node.name === 'FunctionDefinition'  && !node.attributes.isConstructor) var el = yo`<div class=${css.gasEstimation}> <img class=${css.gasStationIcon} title='Estimated gas price' src='https://png.icons8.com/gas-station/win8/50/000000'> ${estimatedGas}</div>`
```

Comment:
Why ignore it for the constructor?

Reply:
@axic good point, forgot to mention this. Please check the other comment I left for you.

Reply:
@axic I have new version of PR that includes your suggestions. Please take a look if you have time. I would like to test it with a contract that has internal functions, but don't have one. Hope it works, but would need to double check.